app/Http/Controllers/AddSingleStreamToCalendarController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Stream;
+use Spatie\IcalendarGenerator\Components\Calendar;
+
+class AddSingleStreamToCalendarController extends Controller
+{
+    public function __invoke(Stream $stream)
+    {
+        $calendar = Calendar::create()
+            ->name("Larastreamers: watch {$stream->title}")
+            ->description('There is no better way to learn than by watching other developers code live. Find out who is streaming next in the Laravel world.');
```

Comment:
Just noticed that we do not store the description of the stream. Later we should change this.